src/main/java/com/google/sps/python/heatMap.py

```diff
@@ -0,0 +1,207 @@
+import numpy as np
+import json
+popFile = 'datasets/county-population.txt'
+caseFile = 'datasets/by-county-and-date.txt'
+<<<<<<< HEAD
+coordFile = 'datasets/usa_county_wise.txt'
+ignoreLocations = [96, 193, 1863]  # Edge case locations where I know the population is 0 (ex: Princess Cruise Ship)
```

Comment:
create a int to represent these as constant values like
```
PRINCESS_CRUISE= 96
LOCATION_2 = 193
LOCATION_3 = 1863
IGNORED_LOCATIONS = [PRINCESS_CRUISE, LOCATION_2, LOCATION_3]
```

It makes it apparent exactly what is being ignored and follows the style guide for constants

Reply:
Actually use _PRINCESS_CRUISE, _LOC ... etc...

The underscore prefix is used to denote that something is internal (private) in python even though python does not have public/private methods, it signals to other developers the intention.